src/WebJobs.Script.WebHost/Security/KeyManagement/DefaultKeyValueConverterFactory.cs

```diff
@@ -23,12 +18,6 @@ public DefaultKeyValueConverterFactory(bool allowEncryption)
 
         private static bool IsEncryptionSupported()
         {
-            if (SystemEnvironment.Instance.IsLinuxContainerEnvironment())
```

Comment:
We want to include this IsLinuxContainerEnvironment check below, so we're doing IsAppServiceEnvironment || IsLinuxContainerEnvironment